spec/lib/importers/npq_manual_validation.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Importers::NPQManualValidation do
+  let(:profile) { create(:npq_validation_data) }
+
+  let(:file) { Tempfile.new("test.csv") }
+
+  around do |example|
+    original_stdout = $stdout
+    $stdout = File.open(File::NULL, "w")
+
+    example.run
+
+    $stdout = original_stdout
+  end
+
+  describe "#call" do
+    subject do
+      described_class.new(path_to_csv: file.path)
+    end
+
+    context "with well formed csv" do
+      before do
+        file.write("application_ecf_id,validated_trn")
+        file.write("\n")
+        file.write("123,7654321")
+        file.write("\n")
+        file.write("#{profile.id},7654321")
+        file.rewind
+      end
```

Comment:
Do we want a fixture for that? (like the files in `spec/fixtures/files`?)

Reply:
I can move it to fixtures but really see the benefit of doing so, that's why I made it inline

Reply:
I think csv files are ugly. That said, in this particular scenario I think keeping them inline makes sense - we want the `profile.id` to be something that is in our database, so it means either constructing the file at runtime (as you do here), or hard-coding the uid, which is not ideal.